### PR TITLE
refactor: switch to qbx lib

### PR DESCRIPTION
**Pull request by TonybynMp4**
## Description

Switched from utils to lib
(why the fuck is the diff looking like that tho)

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: client/properties.lua [hunk 1]
```diff
@@ -220,7 +220,7 @@ local function modifyProperty(propertyData)
     }, function(_, _, args)
         if not args then return end
         if args.action == "name" then
-            local propertyString = string.split(propertyData.name, ' ')
+            local propertyString = string.strsplit(' ', propertyData.name)
```
Comment (D4isDAVID) [inline client/properties.lua]: `string.strsplit` returns each split string individually.

```lua
local one, two = string.strsplit(' ', 'foo bar')

print(one) -- foo

print(two) -- bar
```

File: client/properties.lua [hunk 2]
```diff
@@ -671,7 +671,7 @@ local function openManageMenu(propertyId)
         options = options,
     }, function(_, _, args)
         if args.action == "name" then
-            local propertyString = string.split(propertyData.name, ' ')
+            local propertyString = string.strsplit(' ', propertyData.name)
```
Comment (D4isDAVID) [inline client/properties.lua]: Same here.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]
The diff being weird is probably something with line endings changing(?)

File: client/properties.lua (hunk 1 above)
Comment (TonybynMp4) [reply client/properties.lua]: oh

Comment (D4isDAVID) [reply client/properties.lua]: Actually I noticed you can do this instead
```lua
local propertyString = { string.strsplit(' ', propertyData.name) }
```

Comment (TonybynMp4) [reply client/properties.lua]: i got a better solution

File: client/properties.lua [hunk 3]
```diff
@@ -220,15 +220,14 @@ local function modifyProperty(propertyData)
     }, function(_, _, args)
         if not args then return end
         if args.action == "name" then
-            local propertyString = string.split(propertyData.name, ' ')
-            local propertyNumber = tonumber(propertyString[1])
-            local input = lib.inputDialog(Lang:t('modify_property_menu.title'), {
-                {type = 'input', label = Lang:t('modify_property_menu.name', {name = propertyData.name}), default = table.concat(propertyString, ' ', 2), required = true},
+            local propertyNumber, propertyString = string.strsplit(' ', propertyData.name, 2)
+            local input = lib.inputDialog(locale('modify_property_menu.title'), {
+                {type = 'input', label = locale('modify_property_menu.name', propertyData.name), default = propertyString, required = true},
```
Comment (D4isDAVID) [inline client/properties.lua]: Do the ox locale part in another PR. In this one you didn't change locales to use json files and also didn't include anything in the fxmanifest.

File: client/properties.lua [hunk 4]
```diff
@@ -220,15 +220,14 @@ local function modifyProperty(propertyData)
...
             }, {allowCancel = true})
 
             if input then
                 newData.name = propertyNumber .. " " .. input[1]
-                lib.setMenuOptions('modify_property_menu', {label = Lang:t('modify_property_menu.name', {name = newData.name})}, 1)
+                lib.setMenuOptions('modify_property_menu', {label = locale('modify_property_menu.name', newData.name)}, 1)
```
Comment (D4isDAVID) [inline client/properties.lua]: Same here with the ox locale.

File: client/properties.lua [hunk 5]
```diff
@@ -671,10 +670,9 @@ local function openManageMenu(propertyId)
         options = options,
     }, function(_, _, args)
         if args.action == "name" then
-            local propertyString = string.split(propertyData.name, ' ')
-            local propertyNumber = tonumber(propertyString[1])
-            local input = lib.inputDialog(Lang:t('manage_property_menu.manage_name'), {
-                {type = 'input', label = Lang:t('manage_property_menu.name', {name = propertyData.name}), default = table.concat(propertyString, ' ', 2), required = true},
+            local propertyNumber, propertyString = string.strsplit(' ', propertyData.name, 2)
+            local input = lib.inputDialog(locale('manage_property_menu.manage_name'), {
+                {type = 'input', label = locale('manage_property_menu.name', propertyData.name), default = propertyString, required = true},
```
Comment (D4isDAVID) [inline client/properties.lua]: Same here with the ox locale.

File: client/properties.lua (hunk 3 above)
Comment (D4isDAVID) [inline client/properties.lua]: `default = propertyString` doesn't match the previous way this worked, did you purposefully not include `propertyNumber`? Was including it a bug?

File: client/properties.lua (hunk 5 above)
Comment (D4isDAVID) [inline client/properties.lua]: Same here with the `default = propertyString` part.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: client/properties.lua (hunk 3 above)
Comment (TonybynMp4) [reply client/properties.lua]: It worked the same, it would split the whole string in a table, then for the default value, concatenate it all into a string again ***starting from the second value***, so it would skip the number.

Comment (TonybynMp4) [reply client/properties.lua]: You only modify the name of the property not the number (since that's the db id, could be useful for staff to easily identify it, i guess)

**Review by D4isDAVID — APPROVED** [review]
Seems fine.

**Review by BerkieBb — APPROVED** [review]
